kernel/Makefile.dist.install

```diff
@@ -0,0 +1,19 @@
+# Installation variables
+DESTDIR :=
+DEFAULTSDIR := /usr/share/
+CCDIR := clear-containers
```

Comment:
We might want to use this in kata, so how about making this `PROJECT_DIR` or `REPO_DIR` or something like that?